packages/plugin-typescript/README.md

```diff
@@ -6,13 +6,13 @@
 
 🕵️ **Code PushUp plugin for measuring TypeScript quality with compiler diagnostics.** 🔥
 
-This plugin allows you to **incrementally adopting strict compilation flags in TypeScript projects**.
+This plugin allows you to **incrementally adopt strict compilation flags in TypeScript projects**.
 It analyzes your codebase using the TypeScript compiler to detect potential issues and configuration problems.
 
 TypeScript compiler diagnostics are mapped to Code PushUp audits in the following way:
 
-- `value`: The number of issues found for a specific TypeScript configuration option -> 3
-- `displayValue`: The number of issues found -> 3 issues
+- `value`: The number of issues found for a specific TypeScript configuration option (e.g. 3)
+- `displayValue`: The number of issues found (e.g. "3 issues')
```

Comment:
I mistakenly suggested mismatched quotes :man_facepalming: 

```suggestion
- `displayValue`: The number of issues found (e.g. "3 issues")
```

Reply:
how dare you ;) Thanks for the change request!